XRTK-Core/Packages/com.xrtk.core/Interfaces/InputSystem/IMixedRealityFocusProvider.cs

```diff
@@ -29,13 +29,6 @@ public interface IMixedRealityFocusProvider : IMixedRealitySourceStateHandler, I
         /// <remarks>Every uGUI <see cref="Canvas"/> in your scene should use this camera as its event camera.</remarks>
         Camera UIRaycastCamera { get; }
 
-        /// <summary>
-        /// To tap on a hologram even when not focused on,
-        /// set OverrideFocusedObject to desired game object.
-        /// If it's null, then focused object will be used.
-        /// </summary>
-        GameObject OverrideFocusedObject { get; set; }
```

Comment:
Removed because?

Reply:
It's not needed, the modal input should be used instead.